src/physics/em/detail/MollerBhabhaInteractor.i.hh

```diff
@@ -54,6 +57,15 @@ CELER_FUNCTION MollerBhabhaInteractor::MollerBhabhaInteractor(
 template<class Engine>
 CELER_FUNCTION Interaction MollerBhabhaInteractor::operator()(Engine& rng)
 {
+    if (secondary_energy_cutoff_ >= inc_energy_)
+    {
+        Interaction result;
+        result.action    = Action::unchanged;
+        result.energy    = units::MevEnergy{inc_energy_};
+        result.direction = inc_direction_;
+        return result;
```

Comment:
Can you move this into a helper function `Interaction::from_unchanged`, and add a note inside the `if` here summarizing our discussion about secondary production thresholds?

Reply:
Originally I thought about creating said function, but different interactors have different constructors, so if we create a `from_unchanged(Interaction& in)`  we would have to create the interaction (in this case using `inc_energy/direction_`) and then pass it to `from_unchanged(...)`, which seems more work than just returning it. If we create a `from_unchanged()` with no arguments, then it can only assign `Action::unchanged`, and we will still need to assign the rest of the data before returning the result of the interactor. Another option is to use `from_unchanged(MevEnergy energy, Real3 direction)`. Any suggestions?

Reply:
my thought was just what you say at the end there -- `from_unchanged(MevEnergy energy, Real3 direction)` even though it won't do much will at least ensure it's consistently constructed with a nonzero energy and actual direction -- it'll make it harder to accidentally forget to assign it, and it'll make the code in `MBInteractor` shorter.